### DISCUSSION TITLE
Running this as a Serverless Project Inside a <div> of my Main Project

### DISCUSSION THREAD

**Discussion by JackTelford** [DustinBrett/daedalOS#451] [Q&A]
I am currently working on a project where I would like to run this serverless within my main project.
 My goal is to embed this project directly into a <div> element as a JavaScript snippet into my Main Project.

Do you have any suggestions or ideas on how I could achieve this?

Thank you!

**Comment by DustinBrett** [comment]
Hi, I am not sure how easy it would be for the entire app to be in a div without an iframe, but if you can use an iframe then it's easy. The app when built can run along side whatever it is that is serving your other html file which has the div.

**Reply by JackTelford** [reply]
Hey Dustin I have completed 99% of this integration but when I run yarn build I get my out folder and I run index.html server-less no local host I get a blank screen. Have you tried to run the out folder index.html only? This is my last step to my project so hoping you may have some insight to making it run without a local host. Thank you.!

**Reply by DustinBrett** [reply]
It should work but it depends if you are serving the file in a way where it knows the relative location of supporting documents. Look at the DevTools Network/Console to see what the issue is. Probably something isn't loading. The root folder needs to be where everything is but if you open the index.html file from `file:///C:/x/out/index.html` then it will be relative to `C:/`. The same thing happens for me and it means you need to either serve the files on root or you need to use a web server to host files in a way where it treats the host directory as root.

![image](https://github.com/DustinBrett/daedalOS/assets/16656318/b780b35f-5147-4051-a478-8244796dff09)
